modular_skyrat/modules/customization/modules/clothing/clothing.dm

```diff
@@ -19,6 +19,130 @@
 	var/greyscale_config_worn_taur_paw
 	var/greyscale_config_worn_taur_hoof
 
+	var/static/has_taur_sprite = list()
+	var/static/has_taur_snake_sprite = list()
+	var/static/has_taur_paw_sprite = list()
+	var/static/has_taur_horse_sprite = list()
+
+	var/static/has_taur_sprite_suit = list()
+	var/static/has_taur_snake_sprite_suit = list()
+	var/static/has_taur_paw_sprite_suit = list()
+	var/static/has_taur_horse_sprite_suit = list()
```

Comment:
It's not casted as a list, and why is a list named like a boolean?

Reply:
Its technically a list of booleans. The only possible values are 0 and 1.

Reply:
An ASSOCIATIVE list of booleans, excuse me.